ui/main/src/app/services/lightcards/lightcards-feed-filter.service.ts

```diff
@@ -96,6 +104,17 @@ export class LightCardsFeedFilterService {
             });
     }
 
+    private computeFilteredAndSearchedLightCards() {
+        combineLatest([this.searchService.getSearchChanges(), this.getFilteredLightCards()])
+            .pipe(map((results) => {
+                return this.searchService.searchLightCards(results[0], results[1]);
+            }))
+            .subscribe((lightCards) => {
+                this.logger.debug('Number of cards visible after searching : ' +  lightCards.length, LogOption.LOCAL_AND_REMOTE);
```

Comment:
"Number of cards visible after **filtering and**  searching"

And remove the log just before to avoid to much log 
```
 this.logger.debug('Number of cards visible after filtering : ' +  lightCards.length ,LogOption.LOCAL_AND_REMOTE);
```